docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -79,7 +90,18 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 ### Step 2: Configure integration
 
-In this step the user needs to provide the path to the mongo db log file configured as part of above steps. Typically the logs are located at the location: `/var/log/mongodb/mongodb.log`.
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, you will configure the yaml file required for MongoDB collection.
+
+Below are the inputs required:
+
+- **`Endpoint (no default)`**. The hostname and port of the MognoDB instance, separated by a colon. (For example: `localhost:27017`.)
+- **`Logs Path`**. Enter the path to the log file for your MognoDB instance.
+- **`username (optional)`**. Enter the MognoDB username.
```

Comment:
For username and password, can we mention something on the lines of mongodb receiver doc : 
**username (optional)** : If authentication is required, the user can with **clusterMonitor** permissions can be provided here.
**password (optional)**: If authentication is required, the password can be provided here.

Reply:
fixed it.